### ISSUE TITLE
define additional unit

### ISSUE THREAD

**Issue by spi43984** [TeXhackse/LaTeX-ZUGFeRD#43]
In zugferd.sty some units are defined.

If I wanted to define an additional one (like with KMT for distances in kilometres from the CEFACT Common Code list) - how can this be done? Let's say use km for KMT like piece for H87?

Any help would be appreciated.

**Comment by TeXhackse** [comment]
Documentation for the unit states:

> If unknown, the value is converted to uppercase and we use the selection directly.
> There is a warning in that case.

Please have a look at that before you ask… 

You can always define additional units which don't raise that warning. I am not sure if it's worth implementing the whole list, as it's too specific in general and noone will need the full list.

I guess I might add an option to disable that warning, but in any case it's just a warning.

**Comment by spi43984** [comment]
Yes, I saw that that it raises a warning but nevertheless gets written into the XML. Am just interested to understand how the whole thing works.

**Comment by spi43984** [comment]
I helped myself by copying and extending the definition from zugferd.sty to zugferd-invoice.sty but I don't like that approach as it might break with the next update. Isn't there a more elegant approach to define additional units?

**Comment by TeXhackse** [comment]
It will not break as long as you use `\keys_define` in case that's defined before you would overwrite it. 

I will add an option to disable the warning. But generally the alternative would be to define all units by default and I don't think it's helpful.
